scripts/spellson.lic

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
+
+#
+#
+#   A reworking of Lichmeister Tillmen's inactivespells script
+#   All due credit to concept and vision to Tillmen (tillmen@lichproject.com)
+#
+#   SpellsOn is a transient script.  It will ultimately be renamed
+#   and placed in the Lich scripts repository.  The heart of the functionality
+#   will remain.
+#
+#   Improvements from Tillmen's inactivespells
+#   1) Crafted for GTK3 graphics framework
+#   2) Specifically orders spells in ascending order based on spell number
+#   3) Permits differentiation in progress bar colors
+#
+#   Other improvements planned:
+#   1) Refactor code (get rid of eval and find better balance) - done
+#   2) Enable user personalization of colors, fonts
+#   3) Enable feature to cast from the window
+#   4) A new name
+#
+#   Tillmen's original comments below:
+#
+#   Shows all your inactive spells in a window.
+#
+#   tillmen@lichproject.org
+#
```

Comment:
Probably should turn this into an `=begin`/`=end` block with proper ;repo headers for version, author, etc so it can be version controlled easier on the repository. I'd suggest:
```ruby
=begin
  A reworking of Lichmeister Tillmen's inactivespells script
  All due credit to concept and vision to Tillmen (tillmen@lichproject.com)


        author: elanthia-online
  contributors: Tillmen, Doug
          game: gs
          tags: spells, debuffs, cooldowns, buffs, wizardfe, uberspells
       version: 1.0.0
      required: Lich >= 5.0

     Help Contribute: https://github.com/elanthia-online/scripts
     
  Version Control:
    Major_change.feature_addition.bugfix
  v1.0.0 (2023-06-28)
    - Initial release
    - Rework of Tillmen's inactivespells script
    - Add GTK3 support
    - Ordered spells in ascending order based on spell number
    - Permit differentiation in progress bar colors

  ToDo:
    1) Refactor code (get rid of eval and find better balance) - done
    2) Enable user personalization of colors, fonts
    3) Enable feature to cast from the window

=end
```